External/FEXCore/include/FEXCore/Core/CoreState.h

```diff
@@ -98,8 +98,8 @@ namespace FEXCore::Core {
     uint64_t fs_cached;
```

Comment:
Would be nice if these + gdt could be zero initialised as well, I have a patch for it in my wow64 patchset but you may as well do here

Reply:
Zero initialized the segment members